git-sync/ssh.md

```diff
@@ -0,0 +1,64 @@
+# Using SSH with git-sync
+
+Git-sync supports using the SSH protocol for pulling Git content.
+
+For now, only keys without password protection are supported.
```

Comment:
it seems like password protection is unlikely to provide any value beyond storing the key itself in kube secrets.  Unless you provided a separate password insertion mechanism.